src/net.cpp

```diff
@@ -3286,6 +3288,14 @@ bool CConnman::InitBinds(const Options& options)
         if (!Bind(ipv4_any, BF_REPORT_ERROR, NetPermissionFlags::None)) {
             return false;
         }
+
+        struct in_addr onion_service_target;
+        onion_service_target.s_addr = htonl(INADDR_LOOPBACK);
+        const uint16_t onion_port = GetListenPort() + 1;
+        const CService onion_addr = {onion_service_target, onion_port}; // 127.0.0.1
```

Comment:
Might be missing something, but it seems like this could be simplified:

```c++
const CService onion_addr{DefaultOnionServiceTarget(GetListenPort() + 1);
```